pkgs/development/python-modules/jpylyzer/default.nix

```diff
@@ -2,24 +2,28 @@
 , fetchFromGitHub
 , buildPythonPackage
 , six
+, pytest
 }:
 
 buildPythonPackage rec {
   pname = "jpylyzer";
-  version = "1.18.0";
+  version = "2.0.0";
 
   src = fetchFromGitHub {
     owner = "openpreserve";
     repo = pname;
     rev = version;
-    sha256 = "0vhrq15l6jd5fm6vj7mczjzjpl2ph1dk8jp89dw4vlccky8660ll";
+    sha256 = "01wfbb1bgby9b7m6q7483kvpyc1qhj80dg8d5a6smcxvmy8y6x5n";
   };
 
   propagatedBuildInputs = [ six ];
 
-  # there don't appear to be any in-tree tests as such, but the builder's automatic
-  # runner seems to be upset by the project layout
-  doCheck = false;
+  checkInputs = [ pytest ];
```

Comment:
Please use pytestCheckHook.

Reply:
lol. I don't know if it even existed when I opened this PR.

Reply:
Done & rebased too.